ldi-orchestrator/ldio-connectors/ldio-ldes-client-connector/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioLdesClientConnectorAutoConfig.java

```diff
@@ -40,8 +42,8 @@ public static class LdioClientConnectorConfigurator implements LdioInputConfigur
 		public static final String PROXY_URL_REPLACEMENT = "proxy-url-replacement";
 		private final ApplicationEventPublisher eventPublisher;
 		private final ObservationRegistry observationRegistry;
-		private final RequestExecutorFactory requestExecutorFactory = new RequestExecutorFactory();
-		private final RequestExecutor baseRequestExecutor = requestExecutorFactory.createNoAuthExecutor();
+		private final RequestExecutorFactory requestExecutorFactory = new RequestExecutorFactory(false);
+		private final RequestExecutor baseRequestExecutor = requestExecutorFactory.createNoAuthExecutor(new ArrayList<>());
```

Comment:
Why is this empty list suddenly added as a param? Doesn't the original line do the same anymore?

Reply:
Same story as this morning, I did something else first when this made sense, reverted the changes and forgot to revert this :smile_cat: